config/initializers/sentry.rb

```diff
@@ -4,5 +4,6 @@
   config.breadcrumbs_logger = [:active_support_logger]
   config.capture_exception_frame_locals = true
   config.server_name = ENV['APPLICATION_HOSTNAME'] if ENV['APPLICATION_HOSTNAME'].present?
+  config.excluded_exceptions -= ['ActiveRecord::RecordNotFound']
   config.excluded_exceptions += ['ErrorNotifier::IgnoredError']
```

Comment:
I find it super hard to see that we *remove* something here.

```suggestion
  unignored_exceptions = ['ActiveRecord::RecordNotFound']
  ignored_exceptions = ['ErrorNotifier::IgnoredError']

  config.excluded_exceptions -= unignored_exceptions
  config.excluded_exceptions += ignored_exceptions
```